core/controllers/base.py

```diff
@@ -291,6 +312,13 @@ def render_json(self, values):
     def render_template(
             self, filename, iframe_restriction='DENY',
             redirect_url_on_logout=None):
+        """Renders a template file
+
+        Args:
+            filename: str. Name of the template file
```

Comment:
I did a brief check and I'm seeing things like `self.render_template('pages/collection_player/collection_player.html')`. Doesn't that mean it's a filepath?

Perhaps we should change the name of this variable to `filepath` throughout, and write:

    filepath: str. The path to the template file.